tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -965,6 +971,23 @@ describe("Test Application deployment based on 'clusterGroupSelector'", { tags:
           }
         )
 
+        // Add same label on third cluster
+        if (qase_id === 32) {
+          cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+          cy.contains('.title', 'Clusters').should('be.visible');
+
+          // Add label to the third cluster
+          cy.assignClusterLabel(dsThirdClusterName, key, value);
+
+          // Check application deployed to third cluster
+          cy.wait(500);
+          cy.checkApplicationStatus(appName, dsThirdClusterName, 'All Namespaces');
+
```

Comment:
Sugestion for this or other pr: maybe add negative assertion not being deployed on other clusters

Reply:
Updated checkApplicationStatus with present: true by default to check application present. Changing present option to false will check it's absence/removal.

I have added such a check where is applicable.

Reply:
Sorry, not sure I see where the negative check is done on other cluster. Maybe for another pr?

Reply:
Here I have added it.
https://github.com/rancher/fleet-e2e/pull/236/files#diff-41e30f824d50a402bda9f8b58bf8562c3e9f3ac6fbb1fb193266573fb60a9a45R974-R976

Reply:
that it is not what I meant but pls dont want to delay this anymore. 
feel free to merge